backend/danswer/connectors/github/connector.py

```diff
@@ -42,8 +43,8 @@ def _convert_pr_to_document(pull_request: PullRequest) -> Document:
         sections=[Section(link=pull_request.html_url, text=full_context)],
         source=DocumentSource.GITHUB,
         semantic_identifier=pull_request.title,
+        doc_updated_at=pull_request.updated_at.replace(tzinfo=timezone.utc),
```

Comment:
I'm assuming `pull_request.updated_at` is timezone unaware (but it is UTC)? Can we add a comment about this?